include/vrv/toolkit.h

```diff
@@ -111,7 +111,7 @@ class Toolkit {
      * Creates a midi file, opens it, and writes to it.
      * currently generates a dummy midi file.
      */
-    bool RenderToMidiFile(const std::string &filename);
```

Comment:
I see it more as a rendering than a saving. Why would you save as MIDI ;-)? It does not really matter, but since we have renderToMidi it is introducing even more confusion to change it to SaveMidiFile.